src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -249,10 +264,21 @@ export const canShow = async (
 		hasRequiredConsents(),
 	]);
 
+	await maybeWipeUserData(apiKey as string, brazeUuid);
```

Comment:
It looks to me like this line won't be reached if Braze is switched off. I can see arguments both ways on this one, so I just want to check that this is the behaviour we expect here.

Reply:
Discussed offline but summarising her: yep it's true and I think there are arguments either way here so I'll stick with this for now.